### PR TITLE
Add qa-core Slack channel

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

**Comment by matks** [comment]
ping @SimonGrn

File: slack/channels.md [hunk 1]
```diff
@@ -90,6 +90,13 @@ Please note that like any open source project, the main language on this Slack i
 - Status: Public
 - Language: EN
 
+**#qa-core**
+- Description: For core maintainers and QA team, to discuss QA topics
```
Comment (PierreRambaud) [inline slack/channels.md:94]: I really still don't understand why we reference private channels here :thinking:

Comment (matks) [reply slack/channels.md:94]: I dont know 🤔 I'm just following the rules

Comment (PierreRambaud) [reply slack/channels.md:94]: ping @eternoendless @ttoine Is there a reason?

**Comment by ttoine** [comment]
For transparency purpose.
When project used IRC, it was possible to list all channels, public and private. The aim is to show to people that some work is being done on private channels, even if they can't read it. And also, that explains why they can't find some discussions on public channels.
The issue is that Slack doesn't list private channels when you are not a member. So, there would be no way for people to actually know that there are private channels where work is being done.
Last point: knowing that those channels exist is a proof of seriousness of the project for new comers (they will understand that not all is public) and might a be a motivation for people who want to contribute seriously (they will maybe join some private channels in the future)

**Comment by ttoine** [comment]
Note that if we would respect transparency high standards of open source, all the content of the maintainers channels, and other private channels, should be made available on a website, like the Linux Kernel Mailing List.

**Comment by matks** [comment]
I consider @ttoine as an approval

**Pull request merged**